.github/workflows/binaries.yml

```diff
@@ -42,7 +42,7 @@ jobs:
             builder: ubuntu-20.04
           - target:
               os: macosx
-            builder: macos-12
+            builder: macos-14
```

Comment:
from what I remember of github action runners, this will switch it to arm64

Reply:
Ah good to know, we will need to ship universal then